### ISSUE TITLE
Precompiled app package is not signed

### ISSUE THREAD

**Issue by Olivier-OH** [AlexStrNik/Browserino#6]
By default, MacOS won't open an app that is not signed correctly. 

It could be nice to have a signature so as to make Browserino ready-to-use out of the box.

(in the meantime, i'll allow it to run through the security preferences, of course)

**Comment by AlexStrNik** [comment]
Yeah i know, this also prevents me to publish brew package. Unfortunately in my country there is no easy way to buy Apple Developer Account. But i still plan to do it later this year

**Comment by Olivier-OH** [comment]
Ah, thanks for the feedback! That's not an urgent matter, though, there's always a workaround fortunately. Thanks for the app, also, it's very handy.

**Comment by byt3m4st3r** [comment]
Hi @AlexStrNik,

in the meanwhile you could create a repo named "AlexStrNik/homebrew-Browserino" with a homebrew cask-file.

This will for sure not solving the gatekeeper problem, but the installation could then be easily done with:

`brew tap  AlexStrNik/Browserino`
`brew install Browserino`

And, even updates should be recognized with `brew update` through string manipulations (version methods).

If you once have your developer account, only the already defined cask-file have to be moved to the official homebrew-cask repo.

If you will, I can create the cask-file and a PR for you, if you provide the repo.

Allowing the app, beside the graphical settings menu, can be additionaly done through the terminal with the following command:
`xattr -d com.apple.quarantine /Applications/Browserino.app`

Related info:
- https://docs.brew.sh/Taps
- https://docs.brew.sh/Adding-Software-to-Homebrew#writing-the-cask

**Comment by AlexStrNik** [comment]
Finally found a time to make a brew tap

```bash
brew tap AlexStrNik/Browserino
brew install browserino --no-quarantine
```

**Comment by symbolboxer** [comment]
Homebrew has [removed the `--no-quarantine` flag](https://brew.sh/2025/11/12/homebrew-5.0.0/). Now if you update the Browserino package, macOS will stop you from launching it and offer to move it to the trash.

In the short term I was able to circumvent by running `xattr -cr /Applications/Browserino.app`.

**Comment by lolrenx** [comment]
hi, resurrecting this issue as my company has now disallowed unsigned apps as global policy. any chance to get browserino signed?

**Comment by AlexStrNik** [comment]
I bought the Apple Developer Program a few days ago, so as soon as they approve access I’ll be able to sign the app. It took a while, yeah, but I finally managed to change the country of my Apple account and provide every required ID, document, and whatever else Apple Support asked for...

**Comment by xa4hf8** [comment]
as linked in my previous post, there is a [handy GUI local signing tool](https://itsalin.com/appInfo/?id=sentinel)

**Comment by nirvdrum** [comment]
@AlexStrNik I'm sorry to be a bother, but has there been any more progress? The last release was ~6 months ago and isn't signed.
